utils/types/QuickstartMutationVariable.ts

```diff
@@ -45,7 +45,7 @@ export interface QuickstartMetaData {
   supportLevel?: QuickstartSupportLevel;
   installPlanStepIds?: string[];
   dashboards?: QuickstartDashboardInput[];
-  dataSourceIds?: string[];
+  dataSources?: string[];
```

Comment:
We will want to leave this as `dataSourceIds` because that's what we submit to nerdgraph

Reply:
```suggestion
  dataSourceIds?: string[];
```